packages/PHPStanStaticTypeMapper/TypeMapper/ClosureTypeMapper.php

```diff
@@ -69,12 +70,16 @@ static function (AstNode $astNode): ?FullyQualifiedIdentifierTypeNode {
     public function mapToPhpParserNode(Type $type, string $typeKind): ?Node
     {
         // ref https://3v4l.org/iKMK6#v5.3.29
-        if ($typeKind === TypeKind::PARAM && $this->phpVersionProvider->isAtLeastPhpVersion(PhpVersionFeature::ANONYMOUS_FUNCTION_PARAM_TYPE)) {
+        if ($typeKind === TypeKind::PARAM && $this->phpVersionProvider->isAtLeastPhpVersion(
```

Comment:
if these changes are just CS changes, I would land them before this PR, so we get this one focused on the actual container change